plugins/Live/tests/Integration/ModelTest.php

```diff
@@ -197,20 +197,13 @@ public function test_makeLogVisitsQueryString()
                 $minTimestamp = false,
                 $filterSortOrder = false
         );
-        $expectedSql = ' SELECT  sub.* FROM
-                (
-                    SELECT log_visit.*
+        $expectedSql = ' SELECT log_visit.*
```

Comment:
that's how it simplifies the queries when no segment is applied. No more subquery, and we don't even need a group by anymore.